src/5e-SRD-Subclasses.json

```diff
@@ -45,80 +45,89 @@
 	],
 	"spells": [{
 		"prerequisites": [{
-			"url": "/api/classes/cleric/levels/1",
-			"type": "level"
+			"name": "Cleric 1",
```

Comment:
Is it worth adding this name on the Levels object as well?

Reply:
to have the same schema as prerequisite features and it'd be easier to show just the prerequisites type and name